src/djlint/formatter/utils/tag.py

```diff
@@ -87,14 +101,89 @@ def type(self, val: str) -> None:
 
     @property
     def is_html(self) -> Optional[str]:
+        """
+        Describe if Tag is an HTML element (as defined from Tag argument).
+
+        This is more of a shortand since this method does not make any assertion and
+        only returns what have been set in ``Tag._is_html`` instance attribute. On
+        default, a Tag is not assumed as an HTML element, you need to set it yourself
+        on the instance attribute.
+
+        TODO: Despite return is typed to string, you may be able to just set instance
+        attribute to ``True`` to qualify element as HTML.
+
+        Returns:
+            string: The value of ``Tag._is_html`` instance attribute. It may be
+            ``False`` if attribute has never be explicitely set.
+        """
         return self._is_html
 
     @is_html.setter
     def is_html(self, val: str) -> None:
+        """
+        Setter method to define value of ``Tag._is_html`` attribute.
+
+        TODO: Despite typed as a string, the practical usage of is_html is boolean so
+        it should be safe enough to type it as a boolean.
+
+        Arguments:
+            val (string): Value to set.
+        """
         self._is_html = val
 
+    def _spaceless_left(self) -> str:
+        """
```

Comment:
I recently learned from another issue that jinja also has a "+" option too. But that can be implemented later.